utils/duration_test.go

```diff
@@ -0,0 +1,63 @@
+package utils_test
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"go.viam.com/test"
+
+	"go.viam.com/rdk/utils"
+)
+
+func FuzzDurationJSONRoundtrip(f *testing.F) {
```

Comment:
just to be clear, no special command are required to run this test - all seed inputs (i.e. the `f.Add(input)` lines) will be tested along with any generated input ([example](https://github.com/viamrobotics/rdk/pull/4511/files#diff-d58ec87018d1433c0d2d9c5d8cafb91df7df5c0b11e159d216db471f322daa73)).

new generated inputs might be created by running `go test go.viam.com/rdk/utils -fuzz=FuzzDurationJSONRoundtrip -fuzztime=5m` (the timeout is optional). only inputs that result in a failing test actually result in a new generated input AFAICT - the idea being that failures represent regressions that should be tested against moving forward.